src/module_3/general_functions.py

```diff
@@ -0,0 +1,102 @@
+from pathlib import Path
+
+import pandas as pd
+import numpy as np
+import os
+import joblib
+
+from sklearn.linear_model import LogisticRegression
+from sklearn.metrics import precision_recall_curve, auc, roc_curve
+from sklearn.preprocessing import StandardScaler
+
+# We only consider the features which we have seen previously that are relevant
+FEATURE_COLS = ['ordered_before', 'global_popularity', 'abandoned_before', 'normalised_price']
+TARGET_COL = 'outcome'
+
+def load_data(file_path):
+    data = pd.read_csv(file_path)
+    return data
+
+
+def get_relevant_orders(df):
+    """"We are only interested in orders with at least 5 products"""
```

Comment:
I'd rather to have a more detailed docstrings (what is the expected input and output) than the comments you have in the code which are not adding too much to the code